redhat/frr.spec.in

```diff
@@ -106,16 +106,16 @@ Requires:       ncurses json-c
 Requires(pre):  /sbin/install-info
 Requires(preun): /sbin/install-info
 Requires(post): /sbin/install-info
-BuildRequires:  texi2html texinfo autoconf patch libcap-devel groff 
+BuildRequires:  texi2html texinfo autoconf automake patch libcap-devel groff
```

Comment:
Silas,

thanks for the pointers. I think i need to do some good testing here. Planning to install minimal systems for build and final rpm installation to verify the requirements. I'm expecting potentially more errors here. Will submit an update after this is done.